chrome-extension/test/configure.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+process.exit();
```

Comment:
+ https://github.com/haystack/tipsy/pull/91#issuecomment-275725578
+ Having a test running for firefox is better than having no test. Fixing the chromium part depends on refreshing the versions in the deps in package.json + rewriting the chromium test with an updated api